site/src/theme/components/index.ts

```diff
@@ -0,0 +1,48 @@
+import { ThemeOptions } from "@mui/material";
+
+import { MuiCssBaseline } from "./MuiCssBaseline";
+import { MuiTypography } from "./MuiTypography";
+import { MuiIconButton } from "./MuiIconButton";
+import { MuiPaper } from "./MuiPaper";
+import { MuiButton } from "./MuiButton";
+import { MuiContainer } from "./MuiContainer";
+import { MuiIcon } from "./MuiIcon";
+import { MuiSkeleton } from "./MuiSkeleton";
+import {
+  MuiListItemButton,
+  MuiListItemIcon,
+  MuiListItemText,
+} from "./MuiListItem";
+import { MuiLink } from "./MuiLink";
+import { MuiCheckbox } from "./MuiCheckbox";
+import { MuiTabs, MuiTab } from "./MuiTabs";
+import { MuiBreadcrumbs } from "./MuiBreadcrumbs";
+import { MuiFormControl } from "./MuiFormControl";
+import { MuiInputLabel } from "./MuiInputLabel";
+import { MuiInputBase } from "./MuiInputBase";
+import { MuiOutlinedInput } from "./MuiOutlinedInput";
+import { MuiTextField } from "./MuiTextField";
+
+export const components: ThemeOptions["components"] = {
+  MuiCssBaseline,
+  MuiTypography,
+  MuiIconButton,
+  MuiPaper,
+  MuiButton,
+  MuiContainer,
+  MuiIcon,
+  MuiSkeleton,
+  MuiListItemButton,
+  MuiListItemIcon,
+  MuiListItemText,
+  MuiLink,
+  MuiCheckbox,
+  MuiTabs,
+  MuiTab,
+  MuiBreadcrumbs,
+  MuiFormControl,
+  MuiInputLabel,
+  MuiInputBase,
+  MuiOutlinedInput,
+  MuiTextField,
+};
```

Comment:
I wonder if we could do:

```suggestion
export const componentThemeOptions: ThemeOptions["components"] = {
  MuiCssBaseline: MuiCssBaselineThemeOptions,
  MuiTypography: MuiTypographyThemeOptions,
  // ...
}
```

A symbol called `MuiCssBaseline` placed into `compoents/MuiCssBaseline` looks too much like a component, which might become a mini-trap for some code autocompletion tasks. Using `import { MuiTypographyThemeOptions } from "./MuiTypographyThemeOptions"` / `import { MuiTypographyThemeOptions } from "./mui-typography-theme-options"` is heavier, but less ambiguous to a reader.

Either way, it’d be also nice to sort symbols alphabetically (there is a VS Code command for that).

Reply:
In light of the changes mentioned in [this thread](https://github.com/blockprotocol/blockprotocol/pull/203#discussion_r802743114), I'm inclined to not order them alphabetically, but group them based on the categories from the MUI website. Have done this here: https://github.com/blockprotocol/blockprotocol/pull/203/commits/be20367564a0569581effc13a48b028cc3d3730b